src/app/styles/_panels.scss

```diff
@@ -0,0 +1,22 @@
+$theme-colors: (
+  "highlight",
+  "primary",
+  "secondary",
+  "success",
+  "info",
+  "warning",
+  "danger",
+  "light",
+  "dark"
+);
+
+$theme-color-variants: ("", "-lighter", "-lightest", "-darker", "-darkest");
+
+@each $color in $theme-colors {
```

Comment:
since we talked about the triviality of these classes, might be worth leaving a comment that they're only used in one place currently